### PR TITLE
Fix avutil dependency

### PR DESCRIPTION
**Pull request by funman**
These files depend on avutil headers, and build will fail if it libavutil is not installed in default prefix

### CODE REVIEW

File: examples/Makefile.am [hunk 1]
```diff
@@ -35,6 +35,7 @@ endif
 if HAVE_AUDIOTOOLBOX
 uplay_LDADD += $(UPIPEOSX_LIBS)
 endif
+uplay_CFLAGS = $(AVUTIL_CFLAGS)
```
Comment (cmassiot) [inline examples/Makefile.am]: Should be $(UPIPEAV_CFLAGS) for consistency.

Comment (funman) [reply examples/Makefile.am]: Do you want me to do UPIPEAV_CFLAGS = $(AVUTIL_CFLAGS) ?

Comment (cmassiot) [reply examples/Makefile.am]: UPIPEAV_CFLAGS should already exist.

Comment (nto) [reply examples/Makefile.am]: It seems there is no way to use `libupipe_av_la_CFLAGS` in examples/Makefile.am, so your suggestion (`UPIPEAV_CFLAGS = $(AVUTIL_CFLAGS)`) looks fine to me.

**Pull request merged**